help/compute/diff-studio.md

```diff
@@ -171,18 +168,32 @@ You can set new values for parameters and change values for functions.
 
 ![Multi-stage model - loop](pics/DiffStudio-loop.gif)
 
-Apply the `#update` feature to obtain acyclic simulation.
-Set the length of the new modeling stage in the mandatory `length` variable.
-Inside you can use any mathematical expression to modify functions and parameters,
-same as for the `#loop` section.
+### Multi-stage model
+
+The multi-stage model is a model where parameters and function values can vary for different
+ranges of argument values.
+
+Add name of the first stage in the `#argument` block:
 
 ```python
-#update:
-  length = 6
-  y = 1
+#argument: t, 1-st stage
+  t0 = 0.01
+  t1 = 15
+  h = 0.01
 ```
 
-Use any valid expression, when specifying the updates of inputs.
+Add the `#update` block. Input name of the stage right after the colon. Set the duration of the new modeling stage in the next line using
```

Comment:
> Input name of the stage right after the colon.

Simplify this. And it's "Enter", not "input". And it should be even simpler anyway :)

Reply:
I've reduced it